tests/parsers/test_vasp.py

```diff
@@ -241,9 +241,17 @@ def test_fail_with_multiple_files(self):
 
         # Make the parser
         with self.assertRaises(InvalidIngesterException) as context:
-            VaspParser('perov_relax_U')
+            VaspParser('perov_relax_U', None)
+
+        # Make the parser, but setting `files` to not include `OUTCAR_newname`
+        acceptable_files = [f for f in os.listdir('perov_relax_U')]
+        acceptable_files.remove('OUTCAR_newname')
+        acceptable_files = [os.path.join('perov_relax_U', f) for f in acceptable_files]
+
+        try:
+            VaspParser('perov_relax_U', acceptable_files)
```

Comment:
Don't you want an `assert False, "Shouldn't have parsed without an OUTCAR"` here?